assets/src/checkout/pages/BamboraPayPage.tsx

```diff
@@ -65,9 +67,40 @@ const mountFields = (
     .mount(`#${ids.expiry}`);
 };
 
+const normalizeBamboraBrand = (brand: Brand): String => {
+  switch (brand) {
+    case "visa":
+      return CardTypeBrand.Visa;
+    case "mastercard":
+      return CardTypeBrand.MasterCard;
+    case "amex":
+      return CardTypeBrand.AmericanExpress;
+    case "diners":
+      return CardTypeBrand.DinersClub;
+    case "discover":
+      return CardTypeBrand.Discover;
+    case "jcb":
+      return CardTypeBrand.Jcb;
+    case "maestro":
+      return CardTypeBrand.Maestro;
+    default:
+      return "";
+  }
+};
+
+const isCardBrandSupported = (
+  gatewayAccount: GatewayAccountFragment,
+  brand: Brand
+): boolean => {
+  return gatewayAccount.cardTypes.some(
```

Comment:
i'd probably just call `normalizeBamboraBrand(brand)` once outside of the array filter.

you can also remove the typed return type `boolean` because it should be inferred by TS

Reply:
addressed.